framework/source/class/qx/test/bom/media/MediaTestCase.js

```diff
@@ -189,7 +189,19 @@ qx.Class.define("qx.test.bom.media.MediaTestCase",
       this.wait();
     },
 
-    testPlayEvent: function()
+    // On safari this test fails, if we are running the
```

Comment:
This is awful that it's required. We'd better add lines here:
```
//
// KLUDGE ALERT!
//
```

Reply:
@derrell feel free to add whatever and wherever you think the comment should be changed or moved. You can do that by editing the PR online. I know that this is not an optimal solution, but it’s th only fix I could find.